gui/main_window.py

```diff
@@ -14,16 +21,39 @@ class DicomViewer(QMainWindow):
 
     def __init__(self):
         super(DicomViewer, self).__init__()
+        self.test_dicom = dicom_reader.get_dicom()
+        self.initUI()
+
+    def initUI(self):
+
+        layout = QHBoxLayout()
+
+        # slider
+        sld = QSlider(Qt.Vertical, self)
+        sld.setGeometry(40, 30, 30, 200)
+        sld.setRange(0, 100) # len(self.test_dicom)
+        sld.valueChanged[int].connect(self.change_value)
+        layout.addWidget(sld)
```

Comment:
Don't use sld. Means nothing, use name slider or something more specific. Maybe use a functions to build widgets?